src/Assetic/Extension/Twig/TwigResource.php

```diff
@@ -32,7 +32,7 @@ public function __construct(\Twig_LoaderInterface $loader, $name)
     public function getContent()
     {
         try {
-            return $this->loader->getSource($this->name);
+            return $this->loader->getSourceContext($this->name);
```

Comment:
this is wrong:

- loaders are not forced to implement ``getSourceContext`` in 1.27
- the ``getContent`` method must return the content of the template, not the Twig_Source object.